src/bid/score.rs

```diff
@@ -541,11 +542,13 @@ mod tests {
             alloc_latest_consensus_step,
         );
         verifier.preprocess(&ck)?;
-        verifier.verify(&proof, &vk, &vec![BlsScalar::zero()])
+        verifier
+            .verify(&proof, &vk, &vec![BlsScalar::zero()])
+            .map_err(|e| e.into())
```

Comment:
You don't need `map_err` just to call `into()`, you can use `?`:
```suggestion
        Ok(verifier.verify(&proof, &vk, &vec![BlsScalar::zero()])?)
```